pages/Alert/alert.html

```diff
@@ -0,0 +1,151 @@
+                        sandbox="allow-scripts allow-same-origin">
+                  </iframe>
+                  <h2 class="alert-close-heading">Alert with Close Button</h2>
+                  <p class="alert-warning-para">alert warning is a good sign when you are doing something wrong or you are using any wrong website etc. it will give you
+                        warning so you can get to know it is not good for you.</p>
+                  <div class="warning-main-container">
+                        <div class="alert-close-button-cont">
```

Comment:
Try refactoring this alert container. I feel there are too many styles being applied that can be minimized.